src/ovp_pipeline/llm_client.py

```
@@ -0,0 +1,69 @@
+    try:
+        from .auto_evergreen_extractor import LiteLLMClient, load_env_file
+        from .llm_defaults import resolve_api_key
+    except ImportError:
+        return None
+
+    if vault_dir is not None:
+        try:
+            load_env_file(Path(vault_dir))
+        except Exception:
+            # Don't fail client creation just because .env was missing.
+            pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)